bin/check-smart-status.rb

```diff
@@ -272,7 +272,16 @@ def find_devices
     # Return parameter value if it's defined
     if config[:devices] != 'all'
       config[:devices].split(',').each do |dev|
-        devices << Disk.new(dev.to_s, '', nil)
+        jconfig = @hardware.find { |h1| h1[:path] == dev }
```

Comment:
is there some specific meaning of `h1` as a var name? If not I'd prefer `d` as in the context its much clearer as to its intent.

Reply:
No specific reason. To be honest, I don't know much about ruby syntax so it's probably just a copy/paste from code below in the file.